stylist/admin.py

```diff
@@ -13,7 +14,10 @@ class StyleAdmin(admin.ModelAdmin):
     list_display = ('id', 'name', 'slug', 'site', 'attrs', 'enabled')
     list_filter = ('site', 'enabled')
     search_fields = ('name', 'slug')
-    actions = [compile_styles]
-
+    
+    def __init__(self, model=Style, admin_site=admin.site) -> None:
+        super().__init__(model, admin_site)
+        if not getattr(settings, 'STYLIST_IGNORE_SASS', True):
```

Comment:
This would be a bit cleaner with the other naming convention and with out so many "nots"